docs/data-sources/groups.md

```diff
@@ -43,6 +43,7 @@ Read-Only:
 - `id` (String) The ID of the Group
 - `is_active` (Boolean) Indicates if the Group is active
 - `name` (String) The name of the Group
+- `security_policy_id` (String) The security policy assigned to the Group.
```

Comment:
@alexmensch is this docs string looks ok?

Reply:
Looks good, just needs to be capitalized since it's a product term.

--> "The Security Policy assigned to the Group."

Reply:
updated docs